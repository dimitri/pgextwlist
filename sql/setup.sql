LOAD 'pgextwlist';
ALTER SYSTEM SET session_preload_libraries='pgextwlist';
ALTER SYSTEM SET extwlist.extensions='citext,earthdistance,pg_trgm,pg_stat_statements';
\set testdir `pwd` '/test-scripts'
ALTER SYSTEM SET extwlist.custom_path=:'testdir';
SELECT pg_reload_conf();
