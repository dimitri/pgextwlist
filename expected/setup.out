LOAD 'pgextwlist';
ALTER SYSTEM SET session_preload_libraries='pgextwlist';
ALTER SYSTEM SET extwlist.extensions='citext,earthdistance,pg_trgm,pg_stat_statements,refint';
\set testdir `pwd` '/test-scripts'
ALTER SYSTEM SET extwlist.custom_path=:'testdir';
SELECT pg_reload_conf();
 pg_reload_conf 
----------------
 t
(1 row)

CREATE ROLE mere_mortal;
CREATE OR REPLACE FUNCTION set_extwlist_extname_from_filename() RETURNS VOID SECURITY DEFINER AS $$
    SET extwlist.extname_from_filename TO true;
$$ LANGUAGE SQL;
